### ISSUE TITLE
discoverWebmentionEndpoint returns nothing if link (in HTML) not enclosed in quotes

### ISSUE THREAD

**Issue by janboddez** [indieweb/mention-client-php#38]
I've got this case where an endpoint ( `<link rel=webmention href=[URL]>` in the page's HTML) is not discovered if not explicitly wrapped in (double?) quotes.

Not yet sure if an issue with this library (although it seems like it could improve in this regard, too) or the mf2/mf2 parser, which I'm including, too ...

**Comment by pfefferle** [comment]
@janboddez is this valid HTML? I think the quotes (single or double) are required.

**Comment by jlelse** [comment]
@pfefferle It is valid: https://validator.w3.org/nu/?doc=https%3A%2F%2Fjlelse.blog%2F

**Comment by pfefferle** [comment]
@jlelse hmmm, you are right... so please forget my comment

(I also checked your page with the Webmention WordPress plugin and it seems to work fine... 😁 )

**Comment by janboddez** [comment]
Yeah, I noticed you're using DOMXPath there, which I might do as well. (Although DOMDocument doesn't always work as intended either, IMHO, it'll probably do in this case.) Still, the regex in _this_ library, and possibly mf2 (which I haven't yet looked into), could probably be improved, too.
